p2p/src/network/scheduler/p2p_network_scheduler_actions.rs

```diff
@@ -107,10 +104,6 @@ pub enum P2pNetworkSchedulerAction {
         addr: ConnectionAddr,
     },
     /// Prune streams.
-    PruneStreams {
```

Comment:
@0xMimir why is this not needed anymore? (I saw there was a TODO to remove the action bellow, but I don't know the context)

Reply:
It is action that is not longer needed it has no reducer and no effects anymore so it can be removed